spring-xsuaa-it/pom.xml

```diff
@@ -64,6 +64,11 @@
 			<groupId>org.springframework.security</groupId>
 			<artifactId>spring-security-oauth2-resource-server</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpclient</artifactId>
+			<scope>provided</scope>
```

Comment:
On the consumer side, i don not understand this addition with scope "provided"

Reply:
same as above `Factory method 'springSecurityFilterChain' threw exception; nested exception is java.lang.NoClassDefFoundError: org/apache/http/client/HttpClient`

Reply:
are all samples and the spring-xsuaa-it updated to certificate-based authn?
if not => this would be an incompatible change
if yes => we need to remove scope "provided" as this would mean the Buildpack provides it, and if not the sample would not run on Cloud Foundry.